cmd/manager/main.go

```diff
@@ -29,34 +29,59 @@ type server struct {
 }
 
 func (s *server) CreateStudy(ctx context.Context, in *api_pb.CreateStudyRequest) (*api_pb.CreateStudyReply, error) {
+	var studyID string
+	var err error
 	if in == nil || in.StudyConfig == nil {
 		return &api_pb.CreateStudyReply{}, errors.New("StudyConfig is missing.")
 	}
-	studyID, err := dbIf.CreateStudy(in.StudyConfig)
-	if err != nil {
-		return &api_pb.CreateStudyReply{}, err
+
+	if in.StudyConfig.JobType != "nas" {
+		//If it is a HP job
+		studyID, err = dbIf.CreateStudy(in.StudyConfig)
+		if err != nil {
+			return &api_pb.CreateStudyReply{}, err
+		}
+		s.SaveStudy(ctx, &api_pb.SaveStudyRequest{
+			StudyName:   in.StudyConfig.Name,
+			Owner:       in.StudyConfig.Owner,
+			Description: "StudyID: " + studyID,
+		})
+
+	} else {
+		//If it is a NAS job
+		studyID = "1"
```

Comment:
Why should we set the study id to 1 here, I am not sure if I understand it.

Reply:
See the commentary :)